packages/runtime/container-runtime/src/opLifecycle/outbox.ts

```diff
@@ -413,7 +426,10 @@ export class Outbox {
 			this.batchRebasesToReport--;
 		}
 
-		this.flushInternal(batchManager);
+		this.flushInternal({
+			batchManager,
+			disableGroupedBatching: false,
```

Comment:
will this respect disableGroupedBatching in general?

Reply:
ahhh, i see. this is kind of odd, but seems fine. maybe a comment

Reply:
Hmm don't think I meant to make this change, it's an artifact from switching to params object